android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
Sorry for this particular change . I was giving some problem, may be particular to me so i have to change the gradle version . I will correct this mistake.